src/firefox/data/scripts/firefox_browser_api.ts

```diff
@@ -56,7 +55,8 @@ class FirefoxBrowserApi implements BrowserAPI {
     port.emit('launchTabIfNotOpen', url);
   }
 
-  public startUsingProxy = (endpoint:net.Endpoint) => {
+  public startUsingProxy = (endpoint:net.Endpoint, bypass :string[]) => {
+    //TODO actually use bypass list
```

Comment:
Perhaps log a warning? I'm sure we'll forget. Let's file an issue to make this work in Firefox, too.